### DISCUSSION TITLE
add search params: "Match case", "Whole word", "Regex"

### DISCUSSION THREAD

**Discussion by dranitski** [josdejong/svelte-jsoneditor#359] [Ideas]
Hello! It would be great if json editor could support search params like "Match case", "Whole word", "Regex"

![image](https://user-images.githubusercontent.com/27373393/168782921-70a5d432-c5dc-47dc-990b-403185c32add.png)

thanks in advance!

**Comment by josdejong** [comment]
Yes, definitely. Help would be welcome.

**Comment by mcstarioni** [comment]
Very wanted feature! Also already a great app, much better than VS Code or PyCharm for big json files.

**Comment by josdejong** [comment]
Moving this idea to the [Discussions > Ideas](https://github.com/josdejong/svelte-jsoneditor/discussions/categories/ideas) section since we are not actively working on it. Help implementing this would be welcome.
